frontstage/views/passwords.py

```diff
@@ -58,7 +58,7 @@ def post_forgot_password():
                                    form=form, data=template_data)
 
         if response.status_code != 200:
-            logger.error('Unable to send password change request')
+            logger.error('Unable to send password change request', status=response.status_code)
```

Comment:
Status code is logged out when we call ApiError so do we need to include it here?